google-cloud-datastore/src/main/java/com/google/cloud/datastore/admin/v1/DatastoreAdminSettings.java

```diff
@@ -117,7 +117,7 @@
  *         RetrySettings.newBuilder()
  *             .setInitialRetryDelayDuration(Duration.ofMillis(500))
  *             .setRetryDelayMultiplier(1.5)
- *             .setMaxRetryDelay(Duration.ofMillis(5000))
+ *             .setMaxRetryDelayDuration(Duration.ofMillis(5000))
```

Comment:
Hmm, I think this should have been migrated automatically since this is an autogenerated file. Did we miss this in the generator or has datastore not been updated in a while?

Reply:
Hey, yes it may have slipped in as a manual change, but the bot pushed https://github.com/googleapis/java-datastore/pull/1671/commits/8f4e43048878a647a3f13c850e7cbc7433d60f90#diff-7d6e6e7ee7a1678cd72a99d79903fd5bd2164167a5f85ac66159d374f10375fd and "confirmed" that this should use the new API

> it probably hasn't been updated in a while.
True